experiments/long_runs/snowy_land.jl

```diff
@@ -492,7 +492,7 @@ for (group_id, group) in
     CairoMakie.save(joinpath(root_path, "$(group_name).png"), fig)
 end
 
-short_names = ["gpp", "swc", "et", "ct", "swe", "si"]
+short_names = ["gpp", "swc", "et", "ct", "swe", "si", "trans", "msf"]
```

Comment:
we could also add these to `short_names_bio`/`short_names_other` above too if we want the heatmaps to be in the bio/other png outputs

Reply:
right now i never look at those plots...but all of these are plotted in "figures.pdf"